restcomm/restcomm.mgcp/src/main/java/org/restcomm/connect/mgcp/IvrEndpoint.java

```diff
@@ -119,7 +119,7 @@ private void stop(Object message) {
         request.setNotifiedEntity(agent);
 
 //        //Ask RMS to notify for Recording Signal
-        if (se.getEvent().getName().equalsIgnoreCase("pr")) {
+        if (se.getEvent().getName().equalsIgnoreCase("pr") || se.getEvent().getName().equalsIgnoreCase("pa") ) {
```

Comment:
this is similar to another PR we just introduced. Is there any other place that needs to be updated in similar way (supporting the "pa" event)?

Reply:
@jaimecasero from the discussion I had with Henrique this would be enough, but I will double check with him